cli-commands/compiler/vyperCompiler/vyperCompiler.js

```diff
@@ -0,0 +1,97 @@
+const dockerCLI = require('docker-cli-js');
+const docker = new dockerCLI.Docker();
+const path = require("path");
+const fs = require("fs");
+const mkdirp = require("mkdirp");
+const find_contracts = require("./../etherlime-contract-sources");
+
+
+const run = async (allFiles, buildDirectory) => {
+
+    for (let i = 0; i < allFiles.length; i++) {
+        try {
+            let filePath = path.normalize(path.relative(process.cwd(), allFiles[i])) //extract pure file path
+            let fileBaseName = path.basename(filePath, ".vy")
+            let fileTimestampStatus = await getFileTimestampStatus(filePath)
+
+            if (!await isFileUpdated(fileBaseName, fileTimestampStatus, buildDirectory)) {
+                return
+            }
+
+            let compiledObject = await compile(filePath)
+            compiledObject = Object.assign({"contractName": fileBaseName}, compiledObject[filePath]) //restructuring the compiled object
+            compiledObject.updatedAt = fileTimestampStatus
+
+            await recordCompiledObject(compiledObject, buildDirectory)
+
+            const displayPath = "." + path.sep + filePath;
+            console.log(`Compiling ${displayPath}...`);
```

Comment:
You are printing the message that you are compiling after the compilation is done. It seems to me that this is incorrect